docs/reference-docs/fork-task.md

```diff
@@ -11,19 +12,19 @@ sidebar_position: 1
 
 A Fork operation in conductor, lets you run a specified list of other tasks or sub workflows in parallel. A fork task is
 followed by a join operation that waits on the forked tasks or sub workflows to finish. The `JOIN`
-task also collects outputs from each of the forked tasks or sub workflows.
+the task also collects outputs from each of the forked tasks or sub workflows.
```

Comment:
This change will now read "The JOIN the task also collects outputs from each of the forked tasks or sub workflows.

The old version was "The `JOIN`task also collects outputs from each of the forked tasks or sub workflows."

I think the old version is correct